src/API/TimesheetController.php

```diff
@@ -590,7 +590,7 @@ public function stopAction(Timesheet $id): Response
      *      required=true,
      * )
      *
-     * @Rest\RequestParam(name="copy", requirements="all|tags|rates|meta|description", strict=true, nullable=true, description="Whether data should be copied to the new entry. Allowed values: all, tags, rates, description, meta (default: nothing is copied)")
+     * @Rest\RequestParam(name="copy", requirements="all|tags|rates|meta|description|billable", strict=true, nullable=true, description="Whether data should be copied to the new entry. Allowed values: all, tags, rates, description, billable, meta (default: nothing is copied)")
```

Comment:
```suggestion
     * @Rest\RequestParam(name="copy", requirements="all|tags|rates|meta|description", strict=true, nullable=true, description="Deprecated since 1.15. All data is copied always.")
```